packages/components/molecules/f-media-element/src/FlexStyles.js

```diff
@@ -0,0 +1,117 @@
+import { MODIFIER_OPPOSITES_RULES_MAP, MODIFIER_RULES_MAP } from './constants';
+
+class FlexStyles {
+    flexDirection = '--row';
+
+    flexReverse = undefined;
+
+    defaultFlexDirection = '--row';
+
+    defaultFlexReverse = '';
+
+    operator = undefined;
+
+    operatorOpposite = undefined;
+
+    mediaSize = undefined;
+
+    /**
+     * Adds the specific part of css class string
+     * @param value
+     * @returns {string}
+     */
+    static returnReverseClassSegment (value) {
+        return value ? '--reverse' : '';
+    }
+
+    /**
+     * Adds the specific part of css class string
+     * @param value
+     * @returns {string}
+     */
+    static returnDirectionClassSegment (value) {
```

Comment:
Super NIT naming thing – we tend to prefix these kind of functions `get` rather than `return`.

Not a hard and fast rule, but I'd probably update these to be `getDirectionClassSegment` and above to `getReverseClassSegment`

Reply:
Yea makes sense happy to update